packages/frontend/web-ui/src/scss/home/_gameList.scss

```diff
@@ -0,0 +1,34 @@
+@use '../base/variables' as variables;
+
+.game-list-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--cg-bkg-primary-light);
+  border: 2px solid var(--cg-bkg-secondary-light);
+  border-radius: 10px;
+  box-shadow: 0 2px 4px 0 rgba(68, 2, 131, 0.5);
```

Comment:
This color should be registered as a color variable. Do we really want to add a new color? We are supposed to work with 3 colors max + white / black / gray and maybe lighter and darker versions of our 3 colors

Reply:
Done! I've changed the color with a color of the our palette.